tools/report-converter/codechecker_report_converter/report/output/html/static/js/buglist.js

```diff
@@ -138,6 +151,18 @@ var BugList = {
     review_col.appendChild(document.createTextNode(data['review-status']));
     row.appendChild(review_col);
 
+    var testcase_col = document.createElement('td');
```

Comment:
I would prefer to use variables declared with `let` as they are less bugprone for the avg. developer. (or use `const` if they are not modified.)